life_server/Functions/MySQL/fn_insertVehicle.sqf

```diff
@@ -5,20 +5,18 @@
     Description:
     Inserts the vehicle into the database
 */
-private ["_query","_sql"];
 params [
-    "_uid",
-    "_side",
-    "_type",
-    "_className",
+    ["_uid","",[""]],
+    ["_side",civilian,[civilian]],
```

Comment:
This isn't ideal, as we can't change for bad input, sideUnknown perhaps? You're still checking _side == "" below